quic/s2n-quic-transport/src/space/session_context.rs

```diff
@@ -401,6 +395,20 @@ impl<'a, Config: endpoint::Config, Pub: event::ConnectionPublisher>
         Ok(())
     }
 
+    fn on_server_name(&mut self, server_name: Option<ServerName>) -> Result<(), transport::Error> {
+        // TODO use interning for these values
```

Comment:
You can remove this Todo. The ServerName struct is already allocated and cheaply cloned.